tests/unit/operators/test_centrifugal_asymmetry.py

```diff
@@ -365,6 +383,14 @@ def test_centrifugal_asymmetry():
         dims=["element", "rho_poloidal", "t"],
     )
 
+    mean_charges = DataArray(
+        data=np.swapaxes(
+            np.tile(np.array([5, 4, 3, 2, 1]), (len(elements), len(t), 1)), 1, 2
+        ),
+        coords=[("element", elements), ("rho_poloidal", rho_profile), ("t", t)],
+        dims=["element", "rho_poloidal", "t"],
+    )
```

Comment:
Tile array and multiply by a factor for each element (tungsten should have higher mean charge than beryllium).

Reply:
Could use `ELEMENTS` (from `datatypes.py`) to choose suitable charge values for a given element - it's max charge rather than mean charge but that's wouldn't make a difference for the tests